### PR TITLE
fix typo

### PR DESCRIPTION
**Pull request by sho0126hiro**
I found a typo and corrected it.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,3 +2,5 @@
 gobang
 gobang.yml
 gobang.log
+
+.idea
```
Comment (TaKO8Ki) [inline .gitignore:6]: I think this hidden directory depends on your environment, so gobang doesn't need this.

**Review by TaKO8Ki — COMMENTED** [review]
Thank you for your contribution! I left a comment.

File: .gitignore (hunk 1 above)
Comment (sho0126hiro) [reply .gitignore]: thanks for your comment. I reverted the commit.

**Review by TaKO8Ki — APPROVED** [review]
LGTM :)

**Pull request merged**